src/plugins/Dashboard/FileCard.js

```diff
@@ -56,9 +56,9 @@ module.exports = class FileCard extends Component {
       {this.props.fileCardFor &&
         <div style="width: 100%; height: 100%;">
           <div class="uppy-DashboardContent-bar">
-            <h2 class="uppy-DashboardContent-title">Editing <span class="uppy-DashboardContent-titleFile">{file.meta ? file.meta.name : file.name}</span></h2>
-            <button class="uppy-DashboardContent-back" type="button" title="Finish editing file"
-              onclick={this.handleClick}>Done</button>
+            <h2 class="uppy-DashboardContent-title">{this.props.i18n('editing')} <span class="uppy-DashboardContent-titleFile">{file.meta ? file.meta.name : file.name}</span></h2>
```

Comment:
we should have a way to insert components into a i18n string. I imagine there must be a language where the `Editing` text should come after the file name. we can think about that later tho.

Reply:
Yep, it’s a good idea, thought of that too. No easy solution with current setup, we could just use a variable, but it’s harder to wrap that. We could also simplify and remove “editing” altogether here, but this case might arise somewhere else.